src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -914,88 +915,6 @@ static grpc_channel_args *build_lb_channel_args(
 static void glb_lb_channel_on_connectivity_changed_cb(grpc_exec_ctx *exec_ctx,
```

Comment:
I think this pre-existing forward declaration can go away now too.

Reply:
You are right! Removed it.